Makefile

```diff
@@ -641,11 +662,32 @@ sub-tag-images-%:
 ###############################################################################
 # Windows packaging
 ###############################################################################
+# Pull the BGP configuration scripts and templates from the confd repo.
+$(WINDOWS_MOD_CACHED_FILES): mod-download
+
+$(WINDOWS_ARCHIVE_ROOT)/confd/config-bgp%: windows-packaging/config-bgp%
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -ec ' \
+        $(GIT_CONFIG_SSH) \
+        cp -r `go list -mod=mod -m -f "{{.Dir}}" github.com/kelseyhightower/confd`/$< $@'; \
+        chmod +w $@
+
+$(WINDOWS_ARCHIVE_ROOT)/confd/conf.d/%: windows-packaging/conf.d/%
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -ec ' \
+        $(GIT_CONFIG_SSH) \
+        cp -r `go list -mod=mod -m -f "{{.Dir}}" github.com/kelseyhightower/confd`/$< $@'; \
+        chmod +w $@
+
+$(WINDOWS_ARCHIVE_ROOT)/confd/templates/%: windows-packaging/templates/%
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -ec ' \
+        $(GIT_CONFIG_SSH) \
+        cp -r `go list -mod=mod -m -f "{{.Dir}}" github.com/kelseyhightower/confd`/$< $@'; \
```

Comment:
The `release-windows-archive`  target is faiilng with:
```
cp: cannot create regular file 'windows-packaging/CalicoWindows/confd/conf.d/blocks.toml': No such file or directory01:19
chmod: cannot access 'windows-packaging/CalicoWindows/confd/conf.d/blocks.toml': No such file or directory01:19
Makefile:675: recipe for target 'windows-packaging/CalicoWindows/confd/conf.d/blocks.toml' failed
```

It looks like the `$(WINDOWS_ARCHIVE_ROOT)/confd/templates` directory doesn't exist.

Reply:
I see. I pushed a new commit to fix it.